### PR TITLE
Get Payment Request Info

### PR DESCRIPTION
**Pull request by marctemp**
https://eaflood.atlassian.net/browse/SFI-2315

### CODE REVIEW

File: app/processing/payment-request/get-completed-payment-request-by-payment-request-id.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const db = require('../../data')
+const { COMPLETED } = require('../../constants/statuses')
+
+const getCompletedPaymentRequestByPaymentRequestId = async (paymentRequestId, transaction) => {
+  return db.paymentRequest.findOne({
+    transaction,
+    lock: true,
```
Comment (AbidemiAdio) [inline app/processing/payment-request/get-completed-payment-request-by-payment-request-id.js:7]: I don't think we need to lock record for read purpose.

File: test/mock-payment-request.js [hunk 2]
```diff
@@ -58,7 +62,8 @@ const submitPaymentRequest = {
     completedPaymentRequestId: 1
   }],
   paymentRequestId: 1,
-  referenceId: REFERENCE_ID
+  referenceId: REFERENCE_ID,
+  status: COMPLETED
 }
 
 module.exports = {
```
Comment (AbidemiAdio) [inline test/mock-payment-request.js:69]: Consider moving file to `mock-components` folder

File: app/processing/payment-request/get-completed-payment-request-by-payment-request-id.js (hunk 1 above)
Comment (marctemp) [reply app/processing/payment-request/get-completed-payment-request-by-payment-request-id.js:7]: Correct -- my naive copy-pasta approach

File: test/mock-payment-request.js (hunk 2 above)
Comment (marctemp) [reply test/mock-payment-request.js:69]: I wouldn't want to move it to `mock-components` as they will get lost in that and loss readability and separation of, "these objects are MADE UP of these components"

A `mock-objects` folder could be the way forward if we have many, may of them and they loss that readability.

For now I think becuz they are only 3/4/5 they're fine where they are -- certainly in future if we get 6/7/8+ then organising under a folder will be the way forward

**Review by AbidemiAdio — DISMISSED** [review]
QED

**Review by AbidemiAdio — DISMISSED** [review]
QED

**Pull request merged**